routes/users.js

```diff
@@ -215,6 +217,113 @@ router.get("/:userId/offerings/inactive", async (req, res) => {
   }
 });
 
+// Get all drilled down details about a postings by the user of the given userId (Use this to get all the offerings pending)
+router.get("/:userId/postings/complete", async (req, res) => {
+  try {
+    const postingsOfUser = await Posting.find({
+      ownerId: req.params.userId,
+      active: true,
+    }).populate("offerings");
+
+    res.status(200).json(postingsOfUser);
+  } catch (err) {
+    res.status(500).json({
+      message: "Error code 500: Failed to process request",
+    });
+  }
+});
+
+// Add review to the user with userName
+router.post("/:userName/reviews", async (req, res) => {
```

Comment:
Just curious, why use the `userName` vs the `userId`?

Reply:
since this is posting to another user (aside from the user that is logged in). I thought it'd be easier for the FE to find the target userName than userId (since userName is visible field). (Like in my mind, it'd be quite difficult to grab info about another user, please correct me on this if I'm wrong)

Reply:
Ah gotcha. That makes sense. I will update the FE to use the username instead 👍